code/graph-algorithms/left_view_binary_tree/left_view_binary.py

```diff
@@ -0,0 +1,44 @@
+# A binary tree node
```

Comment:
Same here

Reply:
Please add comment  /* Part of Cosmos by OpenGenus Foundation */ at the top of every code you contribute to this project  ! 👍